conda/recipes/raft-dask/meta.yaml

```
@@ -61,7 +72,7 @@ requirements:
 
 tests:
   requirements:
-    - cudatoolkit ={{ cuda_version }}
+    - cuda-version ={{ cuda_version }
```

I'm going to commit this fix so CI can try again.
```suggestion
    - cuda-version ={{ cuda_version }}
```